table.go

```diff
@@ -274,6 +277,11 @@ func (t *Table) SetBorders(border Border) {
 	t.borders = border
 }
 
+// Append a separator after the current line
+func (t *Table) AppendRowLine() {
+    t.rowLineIdxs[len(t.lines)-1]++
```

Comment:
If the lines is empty, how this work?

Reply:
Good point, it won't do any harm because it's a mapping that's later ignored (and rowLineIdxs was a map[int]int, not an array), but still worth fixing. What I found while adding more tests, though, is that there's two different row printing methods, printRows() and printRowsMergeCells(), which both need to support the new manually added row separators, I'll update my branch with the changes.